packages/strapi-plugin-upload/admin/src/containers/App/index.js

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
+import { DndProvider } from 'react-dnd';
+import HTML5Backend from 'react-dnd-html5-backend';
 import { Switch, Route } from 'react-router-dom';
 import pluginId from '../../pluginId';
 import HomePage from '../HomePage';
 
 const App = () => {
   return (
-    <Switch>
-      <Route path={`/plugins/${pluginId}`} component={HomePage} />
-    </Switch>
+    <DndProvider backend={HTML5Backend}>
+      <Switch>
+        <Route path={`/plugins/${pluginId}`} component={HomePage} />
+      </Switch>
+    </DndProvider>
```

Comment:
Why do we need a second dndprovider here?

Reply:
Because we need one for the plugin for the content manager.

Reply:
But now that I have created another component for the dnd as suggested it's not needed anymore.